vm/vm.go

```diff
@@ -1250,3 +1251,10 @@ func (vm *VM) Fatal(msg string, fields ...zap.Field) {
 	vm.snowCtx.Log.Fatal(msg, fields...)
 	panic("fatal error")
 }
+
+// GetABI returns the ABI (Application Binary Interface) string for the VM.
+// This ABI string represents the interface of the VM,
+// containing all actions.
+func (vm *VM) GetABI() (abi.VM, error) {
```

Comment:
We don't need to expose this i think, I think `vm.Registry()` lets us do this in the caller

Reply:
That's a good catch!